File: tasks/2025/round1_pp/parkour-pp.py
#!/usr/bin/env python3
# @check-accepted: *
def main():
    N, M = map(int, input().split())

    if N > M:
        print("-1")
        return

    for i in range(N):
        for j in range(M):
            if j < i:
                print("#", end="")
            else:
                print(".", end="")
        print()


if __name__ == "__main__":
    main()

File: tasks/2025/round1_pp/parkour-pp.cpp
// @check-accepted: *
#include <fstream>
#include <iostream>
#include <vector>

using namespace std;

int main() {
    int N, M;
    cin >> N >> M;
    if (N > M) {
        cout << "-1\n";
        return 0;
    }
    for (int i = 0; i < N; i++) {
        for (int j = 0; j < M; j++) {
            if (j < i)
                cout << "#";
            else
                cout << ".";
        }
        cout << endl;
    }
}
